sdk/core/System.ClientModel/src/Internal/CancellationHelper.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.ClientModel.Internal;
+
+internal class CancellationHelper
```

Comment:
This type is copied from Azure.Core.  Tracking this technical debt with https://github.com/Azure/azure-sdk-for-net/issues/41035